cmd/eventlistenersink/main.go

```diff
@@ -83,7 +86,26 @@ func main() {
 	profilingServer := profiling.NewServer(profilingHandler)
 	metrics.MemStatsOrDie(ctx)
 
-	sharedmain.WatchObservabilityConfigOrDie(ctx, configMapWatcher, profilingHandler, logger, EventListenerLogKey)
+	exp := metrics.ExporterOptions{
+		Domain:         "",
+		Component:      strings.ReplaceAll(EventListenerLogKey, "-", "_"),
+		PrometheusPort: 9000,
```

Comment:
Would this only work if the metrics backend is Prometheus (and not one of the other supported types?)

Reply:
Ok Ignore this. I did not know what I was talking about.